File: 3 Arrays/CPP/maxMin.cpp
#include <iostream>
using namespace std;

int findMax(int arr[], int size) {
    int max_value = arr[0];

    for (int i = 1; i < size; i++) {
        if (arr[i] > max_value) {
            max_value = arr[i];
        }
    }

    return max_value;
}

int main() {
    int array1[] = {1, 2, 6, 4, 7, 9, 10};
    int size = sizeof(array1) / sizeof(array1[0]);

    int max_value = findMax(array1, size);
    cout << "Maximum value: " << max_value << endl;

    return 0;
}

File: 3 Arrays/Python/Max_Min.py
#function to find the maximum of a array
import array



def find_max(arr):

	max_value = array1[0]

	for i in range(len(array1)):
		if arr[i] > max_value:
			max_value = arr[i]

	return max_value


array1 = array.array('i', [1,2,6,4,7,9,10])

max_value  = find_max(array1)
print(max_value)


